src/main/java/com/google/sps/daos/DatabaseUserDao.java

```diff
@@ -117,7 +118,7 @@ public void createIfNotExists(String idToken) {
   }
 
   @Override
-  public void joinGameInstance(String idToken, String gameInstanceId) {
+  public String joinGameInstance(String idToken, String gameInstanceId) {
```

Comment:
Document this return value.

Reply:
Done! Let me know if its missing more explanation.